bio_hansel/qc/checks.py

```diff
@@ -11,17 +11,15 @@
 
 
 def is_overall_coverage_low(st: Subtype, df: DataFrame, p: SubtypingParams) -> Tuple[Optional[str], Optional[str]]:
-    if not st.are_subtypes_consistent:
-        return None, None
-    if st.subtype is None:
-        return None, None
-    if not st.is_fastq_input():
-        return None, None
+    if not st.are_subtypes_consistent \
+            or st.subtype is None \
+            or not st.is_fastq_input():
+                return None, None
```

Comment:
This return may be too deeply nested and throw an exception when you try to run it. Also just looks off. 4-spaces only please

Reply:
So the correct formatting should be this? 
```
    if not st.are_subtypes_consistent \
            or st.subtype is None \
            or not st.is_fastq_input():
        return None, None
```

Reply:
Yeah, that should work. Passing it through some PyCharm auto-formatting might help too